src/core/verbs/gobal-verbs-sanitizer.ts

```diff
@@ -0,0 +1,2 @@
+export const sanitizeVerbSelectionList = (a: string[], b: string[]): string[] =>
```

Comment:
Do not use "a" and "b", they have no domain meaning

We could use:

currentVerbSelection, fullVerbList

Usually avoid having variable names like "a", "b", "i"... unless you are using them in a very limites scope (for instance inside a filter)